versions/v1/units.json

```
@@ -10494,5 +10494,124 @@
     },
     "source": "qudt.org",
     "sourceReference": "https://qudt.org/vocab/unit/B" 
+  },
+  {
+    "externalId": "frequency:spm",
+    "name": "SPM",
+    "quantity": "Frequency",
+    "longName": "Strokes per minute",
+    "aliasNames": [
+      "spm",
+      "SPM",
+      "strokes per min",
+      "Strokes per minute",
+      "strokes per minute",
+      "Strokes per min",
+      "Strokes/min",
+      "strokes/min",
+      "strokes/m",
+      "Strokes/m"
+    ],
+    "symbol": "spm",
+    "conversion": {
+      "multiplier": 1.0,
+      "offset": 0.0
+    },
+    "source": "SPE Petrowiki",
+    "sourceReference": "https://petrowiki.spe.org/Sucker-rod_pumping_units"
+  },
+  {
+    "externalId": "frequency:hecto-spm",
+    "name": "hecto-SPM",
+    "quantity": "Frequency",
+    "longName": "Hecto Strokes per minute",
+    "aliasNames": [
+      "hectoSPM",
+      "HectoSPM",
+      "hectospm",
+      "Hectospm",
+      "hecto SPM",
+      "hecto spm"
+    ],
+    "symbol": "spm",
+    "conversion": {
+      "multiplier": 100.0,
+      "offset": 0.0
+    },
+    "source": "SPE Petrowiki",
+    "sourceReference": "https://petrowiki.spe.org/Sucker-rod_pumping_units"
+  },
+  {
+    "externalId": "frequency:hecto-hz",
+    "name": "hecto-HZ",
+    "quantity": "Frequency",
+    "longName": "Hecto hertz",
+    "aliasNames": [
+      "hectoHz",
+      "hectoHZ",
+      "hectohertz",
+      "hecto-Hz", 
+      "hecto-hertz", 
+      "HectoHertz", 
+      "Hecto-Hertz",
+      "hHz"
+    ],
+    "symbol": "hHz",
+    "conversion": {
+      "multiplier": 100.0,
+      "offset": 0.0
+    },
+    "source": "qudt.org",
+    "sourceReference": "https://qudt.org/vocab/unit/HZ"
```

You cannot link hecto hertz to the qudt reference for hertz. In this case, we can accept the following syntax:

```suggestion
    "source": "Custom based on qudt.org - https://qudt.org/vocab/unit/HZ",
    "sourceReference": null
```